sofar/sofastream.py

```diff
@@ -0,0 +1,192 @@
+from netCDF4 import Dataset
+import numpy as np
+
+class SofaStream():
+    """
+    Get specific data from SOFA-file without reading entire file into memory.
+
+    :class:`SofaStream` opens a SOFA-file and retreives only the requested
+    data. To access certain data just append its name to your
+    :class:`SofaStream`-instance, e.g. ``'.GLOBAL_RoomType'``.
+    :class:`SofaStream` uses the namespace of sofar – for more information on
+    available data and its associated names refer to the :ref:`conventions
+    section <conventions_introduction>`. Also make sure to check the examples
+    below on how to use :class:`SofaStream`.
```

Comment:
I think I would be more brief here. The examples are doing a good job of explaining the class
```suggestion
    data. See the examples
    below on how to use :class:`SofaStream`.
```